ClemBot.Bot/bot/cogs/trivia_cog.py

```diff
@@ -0,0 +1,452 @@
+import json
+import logging
+import random
+import aiohttp
+import html
+import asyncio
+import typing as t
+from dataclasses import dataclass
+import discord
+import discord.ext.commands as commands
+from discord.ext.commands.errors import UserInputError
+from bot.utils.converters import trivia_cog_converter
+import bot.extensions as ext
+from bot.consts import Colors
+from bot.messaging.events import Events
+
+log = logging.getLogger(__name__)
+
+
+class TriviaCog(commands.Cog):
+
+    def cog_unload(self):
+        self.session.close()
+
+    def __init__(self, bot):
+        self.messages = {}
+        self.bot = bot
+        self.session = aiohttp.ClientSession(headers={"Connection": "keep-alive"})
+
+    @ext.group(case_insensitive=True, invoke_without_command=True)
+    @ext.long_help(
+        "Use trivia to return a random assortment of 10 trivia questions. React with emojis to submit your answer choice")
+    @commands.cooldown(1, 15, commands.BucketType.user)
+    @ext.short_help(
+        "Returns trivia questions")
+    @ext.example("trivia")
+    async def trivia(self, ctx):
+
+        async with await self.session.get(DEFAULT_URL) as resp:
```

Comment:
This usage of aiohttp is incorrect, it should be `async with self.session.get(...` instead.

https://docs.aiohttp.org/en/stable/#client-example